apps/avatax/package.json

```
@@ -4,7 +4,8 @@
   "scripts": {
     "build": "pnpm generate && next build",
     "dev": "pnpm generate && NODE_OPTIONS='--inspect' next dev",
-    "e2e": "vitest --project e2e --watch=false",
+    "e2e": "vitest --project integration --watch=false",
+    "e2e:watch": "vitest --project integration --watch",
```

issue: Will this work? I don't think we've changed the `project` name in vitest config 🤔 

```suggestion
    "e2e": "vitest --project e2e --watch=false",
    "e2e:watch": "vitest --project e2e --watch",
```

---

Yes, it works. I guess I based it on an old commit